### PR TITLE
Integrated git lab ci

### PR DESCRIPTION
**Pull request by vklohiya**
This CI only includes make prod changes, make verify and make docs are not run by this CI.

### CODE REVIEW

File: build-tools/run-in-docker.sh [hunk 1]
```diff
@@ -16,22 +16,33 @@ CURDIR="$(dirname $BASH_SOURCE)"
 : ${build_img:=${BUILD_IMG_TAG}}
 
 # Need to make the directory before docker, to keep it owned by local user
-srcdir=src/github.com/F5Networks/k8s-bigip-ctlr
-wkspace=${PWD}/_docker_workspace
-mkdir -p $wkspace/$srcdir
-
+srcdir=/build/src/github.com/F5Networks/k8s-bigip-ctlr/
+#wkspace=${PWD}/_docker_workspace
```
Comment (ssurenr) [inline build-tools/run-in-docker.sh]: Could you please clean up the comments?

Comment (vklohiya) [reply build-tools/run-in-docker.sh]: removed the comments

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        
+        docker tag "$IMG_TAG" "$BUILD_IMG_TAG"
+        docker push "$BUILD_IMG_TAG" 
+      fi
+Test-Image:
+  tags: 
+    - docker-executor
+
+  stage: Test
+  image: cisbot/test_centos
```
[the remark this answers is not in the document]
Comment (vklohiya) [reply .gitlab-ci.yml]: As discussed, we will create separate repo for this image

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        export LICENSE_STRICT=false
+      elif echo "$CI_COMMIT_REF_NAME" | grep -Eq '^v[0-9]+\.[0-9]+\.[0-9]*' ; then
+        export CTLR_VERSION=$(echo "$CI_COMMIT_REF_NAME" | awk -F'.' '{print $1"."$2}')
+        export LICENSE_STRICT=true
+      else
+        export CTLR_VERSION=$CI_COMMIT_REF_NAME
+        export LICENSE_STRICT=false
+      fi
+    - export CLEAN_BUILD=true
+    - export BASE_OS=debian
```
[the remark this answers is not in the document]
Comment (vklohiya) [reply .gitlab-ci.yml]: For now we just inherited the travis ci, we will revisit this once the automation is ready for redhat based image

**Pull request merged**